GAMES_LIST.md

```diff
@@ -122,6 +122,8 @@
 | factorio             | Factorio                                         |                                                 |
 | farcry               | Far Cry                                          |                                                 |
 | farcry2              | Far Cry 2                                        |                                                 |
+| farmingsimulator2019 | Farming Simulator 2019                           | [Notes](#farmingsimulator)                      |
```

Comment:
The name of the game should be `Farming Simulator 19`, same for 22.

Reply:
Should the naming of the key also be changed?

Reply:
Yep.

Reply:
(Requesting the name change as seeing on their website that they call the game "Farming Simulator 22", not "Farming Simulator 2022")

Reply:
(same goes for the names in the `games.js` file).